src/Core/Localization/CLDR/LocaleData.php

```diff
@@ -106,6 +106,13 @@ class LocaleData
      */
     protected $currencies;
 
+    /**
+     * All territories, by ISO code.
+     *
+     * @var TerritoryData[]
+     */
+    protected $territories;
```

Comment:
```suggestion
    protected $territories = [];
```
Why not this to remove the null condition?

Reply:
:+1: